### ISSUE TITLE
Empty output when training custom model

### ISSUE THREAD

**Issue by amroghoneim** [JaidedAI/EasyOCR#598]
I am trying to train a custom model for Arabic ID card text recognition. I generated a dataset such as the images below.
![ابتداء](https://user-images.githubusercontent.com/37261960/143012424-7fd0cc06-2940-463b-9618-014b41180d45.png)
![أثارت](https://user-images.githubusercontent.com/37261960/143012500-0e21b2b7-759d-4b59-921a-7e2555cfb4da.png)

I then adjusted the folder structure to include the data and the labels.csv files as needed for training and validation.
I was also able to train the model with the given training script. However, the output for the validation becomes empty as seen below.
![image](https://user-images.githubusercontent.com/37261960/143012310-12f877a7-9864-4257-996f-ea055167ab02.png)

These are the configs I am using
number: '١٢٣٤٥٦٧٨٩٠'
symbol: '.'
lang_char: "أإابتثجحخدذرزسشصضطظعغفقكلمنهوءية "
experiment_name: 'ar_ocr'
train_data: 'all_data'
valid_data: 'all_data/data_val'
manualSeed: 1111
workers: 2
batch_size: 32 #32
num_iter: 100
valInterval: 10
saved_model: '' #'saved_models/en_filtered/iter_300000.pth'
FT: False
optim: False # default is Adadelta
lr: 1.
beta1: 0.9
rho: 0.95
eps: 0.00000001
grad_clip: 5
#Data processing
select_data: 'data_train' # this is dataset folder in train_data
batch_ratio: '1' 
total_data_usage_ratio: 1.0
batch_max_length: 100
imgH: 64
imgW: 300
rgb: False
contrast_adjust: False
sensitive: False
PAD: True
contrast_adjust: 0.0
data_filtering_off: False
# Model Architecture
Transformation: 'None'
FeatureExtraction: 'ResNet'
SequenceModeling: 'BiLSTM'
Prediction: 'CTC'
num_fiducial: 20
input_channel: 1
output_channel: 256
hidden_size: 256
decode: 'greedy'
new_prediction: False
freeze_FeatureFxtraction: True
freeze_SequenceModeling: True

<b>What exactly is the problem here?

**Comment by NinaM31** [comment]
From my experience training on Arabic text took more than 100 epochs to start seeing results.

**Comment by Nikhil-Sawant-141** [comment]
@amroghoneim Train for 50,000 iters if not atleast 20,000. 100 is too low. Also with 0.1 LR
